### PR TITLE
Document how to launch a cluster

### PR DESCRIPTION
**Pull request by priteau**
I had to set up my environment again on the new os-api-client, so I made sure the process was documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+Make sure you have a public key uploaded to OpenStack:
+
+    $ openstack keypair list
+
+Launch a cluster by calling the jasmin.yml playbook, passing a config file as
+Ansible extra variables, and providing a keypair to use to connect to the
+cluster. For example:
+
+    $ ansible-playbook -i inventory jasmin.yml -e @config/minimal.yml -e cluster_keypair=<keypair_name>
```
Comment (oneswig) [inline README.md]: The default names for clusters are a bit risky; if two users in the same project run this command, their instances will conflict.  It's worth documenting that here, perhaps as an additional consideration - add a `-e cluster_name=...`

More generally, perhaps we should include the username in the cluster name by default.

**Review by oneswig — COMMENTED** [review]
Looks good, one small paragraph extra if you can...

File: README.md (hunk 1 above)
Comment (priteau) [reply README.md]: I think we can go without including something unique in the cluster name, as eventually this playbook will only be run via AWX and there will be some way to generate a unique name for each cluster.

**Review by oneswig — APPROVED** [review]
Thanks, LGTM

**Pull request merged**